core/src/main/java/org/apache/accumulo/core/metadata/schema/DataFileValue.java

```diff
@@ -105,7 +105,7 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return size + " " + numEntries;
+    return size + " " + numEntries + " " + time;
```

Comment:
I realize that this adds to the existing pattern - but would it be worth it to use a more standardized toString format that includes the field names? Otherwise, you need to read the code to understand that '12 1000 1234` is 'size: 12, numEntries 1000, time 1234'

Reply:
It would be good to improve the toString.  I am thinking maybe I should not make this change in 2.1 because I didn't analyze how this toString method was used. I made the change when trying to figure out the problem with bulk importing delete keys.  I'll either improve it or not change it after looking into how its used.

Reply:
That class is used in a huge number of places.  I decided to err on the side of caution and not change its toString rather try to analyze everywhere its used.